core/utils.js

```diff
@@ -719,3 +739,43 @@ Blockly.utils.wrapToText_ = function(words, wordBreaks) {
   }
   return text.join('');
 };
+
+/**
+ * Check if 3D transforms are supported by adding an element
+ * and attempting to set the property.
+ * @return {boolean} true if 3D transforms are supported
+ */
+Blockly.is3dSupported = function() {
+  if (Blockly.cache3dSupported_ !== null) {
+    return Blockly.cache3dSupported_;
+  }
+  // CC-BY-SA Lorenzo Polidori
+  // https://stackoverflow.com/questions/5661671/detecting-transform-translate3d-support
+  if (!window.getComputedStyle) {
```

Comment:
Use goog.global instead of window.  NodeJS and non-browser environments don't have window.